pykickstart/commands/zerombr.py

```diff
@@ -46,7 +46,7 @@ def _getParser(self):
         return op
 
     def parse(self, args):
-        (_ns, extra) = self.op.parse_known_args(args=args, lineno=self.lineno)
+        (_, extra) = self.op.parse_known_args(args=args, lineno=self.lineno)
```

Comment:
This will conflict with the import of the translation function _.  I'd prefer to just say:

extra = self.op.parse_known_args(args=args, lineno=self.lineno)[1]